### PR TITLE
FABN-1461 NodeSDK add private data event part 1

### PR DESCRIPTION
**Pull request by harrisob**
Add the low level private data to event handling to the
the block decode and event services.
Scenario testing will be part 2

Signed-off-by: Bret Harrison <beharrison@nc.rr.com>

### CODE REVIEW

File: fabric-common/lib/BlockDecoder.js [hunk 1]
```diff
@@ -698,53 +766,53 @@ function decodeEndorserTransaction(trans_bytes) {
...
+// 			} else {
+// 				data.token_action[token_action.data] = {outputs: tokens};
+// 			}
+// 		}
+// 	} catch (error) {
+// 		logger.error(' Unable to decodeTokenTransaction :: %s', error);
+// 	}
+
+// 	return data;
+// }
```
Comment (andrew-coleman) [inline fabric-common/lib/BlockDecoder.js]: is this function no longer needed?  if not, we should delete it

File: fabric-common/lib/BlockDecoder.js [hunk 2]
```diff
@@ -1624,9 +1692,9 @@ const HeaderType = class {
 			case 3:
 				result = decodeEndorserTransaction(proto_data);
 				break;
-			case 9:
-				result = decodeTokenTransaction(proto_data);
-				break;
+			// case 9:
+			// 	result = decodeTokenTransaction(proto_data);
+			// 	break;
 			default:
```
Comment (andrew-coleman) [inline fabric-common/lib/BlockDecoder.js]: same here

**Review by andrew-coleman — DISMISSED** [review]

File: fabric-common/lib/BlockDecoder.js (hunk 1 above)
Comment (harrisob) [reply fabric-common/lib/BlockDecoder.js]: I left it as I heard rumblings of token coming back, but your right I should just remove it

File: fabric-common/lib/BlockDecoder.js (hunk 2 above)
Comment (harrisob) [reply fabric-common/lib/BlockDecoder.js]: I left it as I heard rumblings of token coming back, but your right I should just remove it

**Review by bestbeforetoday — APPROVED** [review]

**Pull request merged**